src/index.js

```diff
@@ -1,4 +1,17 @@
 (function(Chart) {
+  var isObject = function(obj) {
+    var type = typeof obj;
+    return type === 'object' && !!obj;
+  }
+
+  var dataValue = function(dataPoint) {
+    if (isObject(dataPoint)) {
+      return dataPoint.y;
```

Comment:
I think it needs to support `horizontalBar`.
When chart type is `horizontalBar`, datapoints object has `x` property.

```js
new Chart(..., {
  type: "horizontalBar",
  data: {
    ...,
    datasets: [
      { data: [{ x: 1 }, { x: 2 }, { x: 0 }] },
    ]
  },
  ...
}
```

Reply:
You're right. I'll try to fix it soon, thanks.